doc/COMPILING/COMPILING.md

```diff
@@ -121,6 +124,51 @@ Instructions for compiling on a Debian-based system. The package names here are
 
 The building instructions below always assume you are running them from the Cataclysm:DDA source directory.
 
+# Fedora
+## Ncurses builds
```

Comment:
The Debian header above should get moved down to actually be above the debian instructions, below the Gentoo section.

Reply:
Completely overlooked it, it should be fixed now.